lib/cc/cli/command.rb

```diff
@@ -43,6 +43,10 @@ def require_codeclimate_yml
         end
       end
 
+      def set_local_owner
+        File.chown(uid, gid, filesystem.path_for(CODECLIMATE_YAML))
```

Comment:
Sorry, I think this method should still take an argument, I meant to get rid of the defaulting.